pkg/net/default.go

```
@@ -457,7 +466,98 @@ func moveFiles(b *models.Binaries) error {
+		}
+		root = rendered
+	}
+
+	return filepath.Join(b.DownloadFolder, root)
+}
+
+func discoverExtractedSourcePath(b models.Binaries, relativePath string, strictRelativeMatch bool) string {
+	var match string
+	_ = filepath.WalkDir(b.DownloadFolder, func(path string, d os.DirEntry, err error) error {
+		if err != nil {
+			return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 521) but it returns nil

(nilerr)